vocabulary-draft/introduction.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Delivery Events Vocabulary
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+This document intends to describe a set of events related to different phases of a Continuous Delivery process. 
+These events are categorized by meaning and the phase where they are intenteded to be used. 
+These events are agnostic from any specific tool and are designed to fit a wide range of scenarios. 
+
+The phases covered by this proposal are:
+
+- **Source Code Version Control Events**: Events emitted by changes in source code or by the creation, modification or deletion of new repositories that hold source code.
+- **Continuous Integration Pipelines Events**: includes events related to building, testings, packaging and releasing software artifacts, usually binaries.
+- **Continuous Deployment Pipelines Events**: include events related with environments where the artifacts produced by the integration pipelines actually run. These are usually services running in a specific environment (dev, QA, production), or embedded software running in a specific hardware. 
+
+These phases can also be considered as different profiles of the vocabulary that can be adopted independently. 
+
+# Required Metadata for CD Events
+
+The following attributes are REQUIRED to be present in all the Events defined in this vocabulary:
+
+- **Event ID**: defines a unique identifier for the event
```

Comment:
Should we also state what syntax we expect the defined parameters to fulfill, or should we take all such definitions in later PRs? This comment is valid for all parameters in the recently created PRs.

Reply:
@e-backmark-ericsson yeah.. let's do that in a follow-up PR.